lib/rucio/api/rse.py

```diff
@@ -24,6 +24,7 @@
 # - Gabriele <sucre.91@hotmail.it>, 2019
 # - elichad <eli.chadwick.256@gmail.com>, 2020
 # - patrick-austin <patrick.austin@stfc.ac.uk>, 2020
+# - David Población Criado <david.poblacion.criado@cern.ch>, 2021
```

Comment:
Please use `tools/add_header -i %s` script to add the header. Since you are using a special character in your name, we should make sure that the utf-8 encoding is mentioned at the top of the file (Which the add_header script will do)